extensions/amp-playbuzz/0.1/test/test-amp-playbuzz.js

```diff
@@ -75,6 +75,8 @@ describes.realWin(
       ins.setAttribute('alt', 'Testing');
       if (opt_responsive) {
         ins.setAttribute('layout', 'responsive');
+      } else {
```

Comment:
Is this change related? I'm not following it regarding strict mode and missing `this`?